c/helper.ml

```diff
@@ -28,7 +28,7 @@
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  *)
 
-open Stdext.Xstringext
+open Xapi_stdext_std.Xstringext
```

Comment:
It's used only for the split below, you might as well only `open Xapi_stdext_std` that just exports the module names, and use it in-place on line 35